src/components/organisms/UiMultipleChoices/UiMultipleChoices.vue

```diff
@@ -141,6 +146,13 @@ const updateHandler = (newValue: MultipleChoicesModelValue, index: number) => {
   emit('update:modelValue', value.value);
 };
 
+function focusInvalidChoice(inputs: InvalidInputs) {
```

Comment:
Why does it take param instead of using `invalidInputsRefs`?
I don't think we need to make this flexible, it can work exactly like the watcher.